mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -54,6 +55,12 @@ data class Widget(
     private val rawMinValue: Float?,
     private val rawMaxValue: Float?,
     private val rawStep: Float?,
+    // sub-widgets are for the buttons in buttongrids
+    val widgets: List<Widget>?,
```

Comment:
Why not name the member variable `buttons` to make the comment content more obvious?

Reply:
ok